### DISCUSSION TITLE
Crash while hovering with an item over a crate (nullref exception in IsInventoryHoverAvailable)

### DISCUSSION THREAD

**Discussion by Udrakan** [FakeFishGames/Barotrauma#13618] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I was hovering with a battery over already occupied battery slot in the nitroglycerin crate battery slot and I got ragdolled by water. I couldn't reproduce this crash. Stable.
[crashreport (24).log](https://github.com/Regalis11/Barotrauma/files/13924585/crashreport.24.log)

### Reproduction steps

???

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I could not find a way to reproduce this either, and I can't see anything on that line that should be throwing a nullref exception. I think we might just need to hope for the best and hope we run into this again and manage to collect some more information. :/
